deepmd/pt/model/atomic_model/density_atomic_model.py

```
@@ -0,0 +1,370 @@
+            the result dict, defined by the `FittingOutputDef`.
+
+        """
+        nframes, nloc, nnei = nlist.shape
+        atype = extended_atype[:, :nloc]
+        if self.do_grad_r() or self.do_grad_c():
+            extended_coord.requires_grad_(True)
+        assert mapping is not None
+        assert grid is not None
+        assert grid_type is not None
+        assert grid_nlist is not None
+        bsz, ngrid, nnei = grid_nlist.shape
```

Ruff (0.16.2)

[warning] Unpacked variable `bsz` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)